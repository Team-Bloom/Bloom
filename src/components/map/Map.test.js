import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import {Node, NodeObject} from './'

it('renders Node without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<Node />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it('renders NodeObject without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<NodeObject />, div)
  ReactDOM.unmountComponentAtNode(div)
})

test('node changes when user enters text', () => {
    const component = renderer.create(<Node />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
