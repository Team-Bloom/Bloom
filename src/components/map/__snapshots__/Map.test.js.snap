// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`node changes when user enters text 1`] = `
<div
  className="nodeWrap"
>
  <svg
    className="line"
    height="52px"
    width="52px"
  >
    <line
      style={
        Object {
          "stroke": "rgb(255,0,0)",
          "strokeWidth": 2,
        }
      }
      x1="0"
      x2="50px"
      y1="0"
      y2="50px"
    />
  </svg>
  <div
    className="dragger"
    onMouseDown={[Function]}
    style={
      Object {
        "left": "50px",
        "top": "50px",
      }
    }
  >
    <div
      className="node"
      onClick={[Function]}
    >
      <span>
        Node Name
      </span>
      <svg
        aria-labelledby="title"
        className="nodeIcon"
        viewBox="0 0 32 32"
        width="100%"
      >
        <title
          id="title"
        >
          Node Name
        </title>
        <rect
          height="50%"
          style={
            Object {
              "fill": "rgb(255,255,255)",
              "stroke": "rgb(0,0,0)",
              "strokeWidth": "0.01em",
            }
          }
          width="100%"
        />
      </svg>
      <button
        onClick={[Function]}
      >
        Add 
      </button>
    </div>
  </div>
</div>
`;
